cli/repair/clusterwide_config.go

```diff
@@ -264,6 +264,7 @@ func (topologyConf *TopologyConfType) setReplicasetsConf() error {
 				return fmt.Errorf("Replicaset %s config doesn't contain %q key", replicasetUUID, keyReplicasetRoles)
 			}
 
+			// XXX: empty map is a list!!
```

Comment:
Don't forget to file issues for all "XXX"

Reply:
https://github.com/tarantool/cartridge-cli/issues/449